src/applications/disability-benefits/view-payments/containers/helpers.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export const breadcrumbLinks = [
+  <a href="/" aria-label="Back to VA Home page" key="1">
+    Home
+  </a>,
+  <a
+    href="/view-change-dependents/"
```

Comment:
Should this href hold a different value?

Reply:
Yes but I don't know what the hrefs should be yet, they are still working on an IA